dashboard/dashboard/templates/registration/change_password.html

```diff
@@ -0,0 +1,37 @@
+{% extends "racetrack/base.html" %}
+
+{% block content %}
+
+<div class="container">
+<div class="card">
+<div class="card-body">
+
+    <h5 class="card-title">Change Password</h5>
```

Comment:
Ah, you could also display here password rules, so users upfront know what is the minimum length, special chars etc. without having to go through trial-error.

Reply:
Fortunately, it is displayed out-of-the-box thanks to default Django's password form :smile: 

![Screenshot from 2022-10-25 15-58-44](https://user-images.githubusercontent.com/45496492/197793887-c107b4ac-aa7e-463f-95a6-7627857e7c25.png)

Reply:
Great :) forgot about it.